### DISCUSSION TITLE
Expo Sqlite Next - Mobile Driver

### DISCUSSION THREAD

**Discussion by abdelhameedhamdy** [electric-sql/electric#789] [Questions]
Hey there, happy new year,

Wondering, should the current mobile driver for expo-sqlite work with the new release of [Expo-Sqlite-Next](https://docs.expo.dev/versions/v50.0.0/sdk/sqlite-next) ?

Thanks

**Comment by jljorgenson18** [comment]
I am using expo-sqlite/next on my project and it seems to be working. I couldn't get things working without /next

**Reply by abdelhameedhamdy** [reply]
It seems not, as the current driver refers to  SQLiteDatabase execRawQuery function which is not available currently in /next, wondering, how it works with electric in your project ? 

Update: just to clarify expo-sqlite-next is working, but adopting it with electric-sql is not compatible.

**Reply by jljorgenson18** [reply]
O, my apologies. I think I tried to use sqlite next at one point to solve a different problem but I ended up rolling with expo-sqlite 11.3.3

I was making so many changes to get this all working that I got confused. You are right in that the execRawQuery was the problem.

**Comment by abdelhameedhamdy** [comment]
Hope that electric-sql expo driver get updates soon to adopt the new expo-sqlite-next.

**Reply by balegas** [reply]
Hi, we're excited about adding a driver for `expo-sqlite-next`.

It  is not super complicated and others have went through the steps of creating a [new driver](https://github.com/electric-sql/electric/pull/484) for electric.

In case you're interested we could chat a bit more.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
